fdbcli/AuditStorageCommand.actor.cpp

```diff
@@ -96,19 +96,37 @@ ACTOR Future<UID> auditStorageCommandActor(Reference<IClusterConnectionRecord> c
 			return UID();
 		}
 
-		UID startedAuditId = wait(auditStorage(clusterFile, KeyRangeRef(begin, end), type, /*timeoutSeconds=*/60));
+		KeyValueStoreType engineType = KeyValueStoreType::END;
+		if (tokens.size() == 5 && (type == AuditType::ValidateHA || type == AuditType::ValidateReplica)) {
+			if (tokencmp(tokens[4], "sqlite")) {
+				engineType = KeyValueStoreType::SSD_BTREE_V2;
+			} else if (tokencmp(tokens[4], "rocksdb")) {
```

Comment:
nit: make the storage engine name consistent with those in `configure` command. Also use `KeyValueStoreType::fromString` to convert string to storeType.

Reply:
Yes. Revised.